### DISCUSSION TITLE
Copilot X Chat Beta for Intellij?

### DISCUSSION THREAD

**Discussion by csvan** [community/community#65988] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I was recently granted access to the Copilot X Chat feature via my organisation (Github for Business). Very exciting! Sadly, from what I can see it is only supported in VS Code for now, I was curious if there is a roadmap for supporting it in the IntelliJ plugin?

**Comment by Akash1134** [comment, marked as answer]
Hi @csvan ,

Thank you for posting in the Copilot community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

At this time, the public preview for Copilot for Chat is for VS or VSCode users. We don't have a roadmap for the release supporting other IDEs including Intellij but want to ensure that we're looking into it.

**Comment by csvan** [comment]
Thanks @Akash1134 !
